openrtb_ext/imp.go

```diff
@@ -2,10 +2,11 @@ package openrtb_ext
 
 // ExtImp defines the contract for bidrequest.imp[i].ext
 type ExtImp struct {
```

Comment:
It looks like this struct is unused throughout the project... you can just delete it.

Reply:
ExtImp struct is used in openrtb_ext/imp_rhythmone.go file for adding custom extension object in bid request.